test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/ArchiveConsistency.java

```diff
@@ -152,8 +152,7 @@ private static void doTest(String baseArchiveName, String topArchiveName) throws
 
         runTwo(baseArchiveName, wrongBaseName,
                appJar, mainClass, 1,
-               new String[] {"Base archive " + baseArchiveName,
-                             " does not exist",
+               new String[] {"Base archive name is damaged",
                              "Header checksum verification failed"});
```

Comment:
It would be good if there's a testcase to cover the "does not exist" case.
It's fine if it will be covered by PR #5997.

Reply:
I added a new test case where the base archive doesn't exist.